GitUI/CommandsDialogs/RevisionDiff.cs

```diff
@@ -107,6 +107,16 @@ internal Keys GetShortcutKeys(Commands cmd)
 
         #endregion
 
+        public void RevisionDiff_KeyDown(object sender, KeyEventArgs e)
+        {
+            if (e.KeyCode == firstToSelectedToolStripMenuItem.ShortcutKeys && Visible
+             && !(DiffText.WouldHandle(firstToSelectedToolStripMenuItem.ShortcutKeys) && DiffText.ContainsFocusedControl()))
```

Comment:
The more I look at it, the more I get a feeling that `WouldHandle` means `FileViewer control has handled the shortcut`. Is this the intent? If so, perhaps it is better named `Handled(Keys)` or `HasHandled(Keys)`.